corgi/tasks/prod_defs.py

```diff
@@ -159,8 +161,10 @@ def update_products() -> None:
                                             }
                                         },
                                     )
+                                    # TODO: Why are there two places that create a variant node?
```

Comment:
feel free to rationalise ... there maybe no good reason why (other then we never got to 'three' to boil it down!).